module1.js

```diff
@@ -104,13 +101,9 @@ function question1(name1) {
     "The array contains { number of animals } animals. These are: {animal1} and {animal2} and {animal3}."
 */
 
-// function question6( animArr ) {
-//   // TODO
-
-//   return /* ??? */;
-// }
-
-
+function question6(animArr) {
+  return "The array contains " + animArr.length + " animals. These are: " + animArr.join(' and ') + "."
+}
 /**
```

Comment:
A nice neat solution, and probably shorter than extracting each item from the array with animArr[] syntax and concatenating things individually.